cli/flox/src/commands/environment.rs

```diff
@@ -1091,26 +1091,40 @@ impl Update {
     }
 }
 
-/// upgrade packages using their most recent flake
+/// Upgrade packages in an environment
 #[derive(Bpaf, Clone)]
 pub struct Upgrade {
     #[allow(dead_code)] // pending spec for `-e`, `--dir` behaviour
     #[bpaf(external(environment_args), group_help("Environment Options"))]
     environment_args: EnvironmentArgs,
 
-    #[allow(unused)] // Command currently forwarded
     #[bpaf(external(environment_select), fallback(Default::default()))]
     environment: EnvironmentSelect,
 
-    #[allow(dead_code)] // not yet handled in impl
-    #[bpaf(positional("PACKAGES"))]
-    packages: Vec<String>,
+    #[bpaf(positional("GROUP|INSTALL ID"))]
```

Comment:
suggestion (non-blocking): use lowercase and mention id first

lowercase descriptions are formatted as `<package or group>` while uppercase ones will be `PACKAGE OR GROUP`.
In the latest commands we've been using the `< .. >` style since it looks "friendlier" and formats better (spaces are ambiguous as OR and GROUP could be two more separate arguments).

Non-blocking because we'll go through the args following our style guide recommendations  on this anyway.

suggestion: add argument description

```suggestion
    /// ID of a package or group name to upgrade
    #[bpaf(positional("package or group"))]
```

Reply:
apparently it's already in the style guide: https://www.notion.so/floxdev/flox-CLI-Style-Guide-ff785fd9bdbc445284ab6e1c660984a8?pvs=4#1cb7f18852a34af19534ed816ba81153